docs/docs/usage.md

```diff
@@ -230,6 +230,20 @@ includes:
       DOCKER_IMAGE: frontend_image
 ```
 
+### Namespace aliases
+
+When including a taskfile, you can give the namespace a list of `aliases`. This works in the same way as [task aliases](#task-aliases) and can be used together to create shorter and easier-to-type commands.
+
+```yaml
+version: '3'
+
+includes:
+  generate:
+    taskfile: ./taskfiles/Generate.yml
+    aliases:
+      - gen
```

Comment:
What about using one line arrays for `aliases` mentions on documentation? This would be consistent with `deps` and is shorter.

```suggestion
    aliases: [gen]
```